packages/animations/lib/src/open_container.dart

```diff
@@ -203,6 +205,11 @@ class OpenContainer extends StatefulWidget {
   /// Defaults to [ContainerTransitionType.fade].
   final ContainerTransitionType transitionType;
 
+  /// Whether the root Navigator should be used in the route transition.
+  ///
+  /// Defaults to [false].
+  final bool rootNavigator;
```

Comment:
nit: let's call this `useRootNavigator` like in other parts of Flutter's API, e.g. https://master-api.flutter.dev/flutter/material/showDialog.html

From there you can also copy some of the documentation describing the difference between setting this to true or false.